test/spec/command-monitoring/unified/server-connection-id.json

```diff
@@ -0,0 +1,101 @@
+{
+  "description": "server-connection-id",
```

Comment:
This test should be done as part of NODE-3449 / DRIVERS-1141, I don't think we should sync it here unless we are also ready to update our logic to append `serverConnectionId` when it is available

Reply:
I've removed here.